packages/server/src/queries/blockaid/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from "./transaction";
+
+export const blockaidAuthHeaders: HeadersInit | undefined = process.env
+  .BLOCKAID_API_KEY
+  ? {
+      "x-api-key": process.env.BLOCKAID_API_KEY,
+    }
+  : undefined;
```

Comment:
If the API Key is undefined, I'm assuming it won't cause a breaking error? I'm thinking about third party developers who download the app and want to run it locally but wont have the env variable.

Reply:
Ye they'll get errors, but it should not break anything because we don't await for this promise inside the sign and broadcast steps